tribe-ext-organizer-notifications.php

```diff
@@ -76,18 +76,33 @@ public function generate_email( $other = null, $event_id = null ) {
    }
 
     /**
-     * Get organizer's email address.
+     * Get all organizer's email addresses
      *
      * @param $post_id
-     * @return string
+     * @return array
      */
     private function get_recipient( $post_id ) {
 
-        // Get the organizer's email
-        $to = html_entity_decode( tribe_get_organizer_email( $post_id ), ENT_COMPAT, 'UTF-8' );
+        //get all organizers asociated to the post
+        $organizers_ids = tribe_get_organizer_ids( $post_id );
+
+        $to = array();
+
+        //get the email for each organizer
+        foreach ( $organizers_ids as $organizer_id ) {
+            $organizer_email = tribe_get_organizer_email( $organizer_id, false );
+
+            //make sure it's a valid email
+            if ( is_email( $organizer_email ) ) {
+                $to[] = $organizer_email;
+            }
+        }
 
-        // Return valid email, or empty string
-        return ( is_email( $to ) ) ? $to : '';
+        if ( empty( $to ) ) {
+            return '';
```

Comment:
Match `@return` type
```suggestion
            return [];
```